lib/client-assets.php

```diff
@@ -158,6 +158,12 @@ function gutenberg_register_scripts_and_styles() {
 		) ),
 		'before'
 	);
+	wp_register_script(
+		'wp-viewport',
+		gutenberg_url( 'viewport/build/index.js' ),
+		array( 'wp-element', 'wp-data', 'wp-components' ),
```

Comment:
Just noting that it has too many dependencies to move it to `@wordpress/packages`.